python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S6887.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "pytz.timezone should not be passed to the datetime.datetime constructor",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6887",
+  "sqKey": "S6887",
+  "scope": "All",
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "HIGH",
+      "RELIABILITY": "HIGH",
+      "SECURITY": "LOW"
```

Comment:
This is an action for me but I have to update the rules metadata before we merge. Namely, there shouldn't be a security impact, and we may or may not want to keep both maintainability and reliability impacts.